### PR TITLE
Tweak optional CI run execution.

### PR DESCRIPTION
**Pull request by fruffy**
Try to improve the execution of optional CI runs. You can only really test whether things are working by running this "live".

### CODE REVIEW

**Comment by fruffy** [comment]
@vlstill This fixes the sanitizer complaints you may see.

**Review by jafingerhut — APPROVED** [review]
I am actually not the best person to review most of this, e.g. the .yml files for Github actions are still Greek to me.  But going ahead and approving in belief that this only affects the behavior of the expensive CI runs based upon whether they have appropriate tags on the PRs.

File: backends/bmv2/run-bmv2-ptf-test.py [hunk 1]
```diff
@@ -277,6 +278,8 @@ def run_test(options: Options) -> int:
     switch_proc = testenv.run_simple_switch_grpc(switchlog, grpc_port)
     if switch_proc is None:
         return testutils.FAILURE
+    # Breath a little.
+    time.sleep(1)
```
[the remark this answers is not in the document]
Comment (fruffy) [reply backends/bmv2/run-bmv2-ptf-test.py:282]: I will try to come up with a more sophisticated solution soon. Right now I am not even sure if this helps. Retrying for a bit is likely the best solution.

**Comment by fruffy** [comment]
> I am actually not the best person to review most of this, e.g. the .yml files for Github actions are still Greek to me. But going ahead and approving in belief that this only affects the behavior of the expensive CI runs based upon whether they have appropriate tags on the PRs.

Thanks! I do not think anyone is really qualify to review these. You also only can really test them by merging the files and seeing what happens. There are a few ways to try out actions locally, but exercising and making sure the triggers work is trial and error.

**Pull request merged**